src/main/java/kea/alog/project/domain/topic/controller/TopicController.java

```diff
@@ -26,9 +26,17 @@ public ResponseDto<PageDto<TopicDto>> findAll(
         @RequestParam("sortType") TopicSortType sortType,
         @RequestParam("page") int page,
         @RequestParam("size") int size,
-        @PathVariable Long projectPk
+        @PathVariable("projectPk") Long projectPk
     ) {
         return ResponseDto.success(200,
             topicService.findAll(projectPk, keyword, sortType, page, size));
     }
+
+    @GetMapping("/{topicPk}")
+    public ResponseDto<TopicDto> findOne(
+        @PathVariable("projectPk") Long projectPk,
```

Comment:
If you want to use PathVariable, then url should be "/{projectPk}/{topicPk}

Reply:
Because topic belongs to the project, the topic-related url starts with `/api/projects/{projectPk}/topics`!

Reply:
K! I didnt see @RequestMapping